draft-wang-ppm-dap-taskprov.md

```diff
@@ -213,6 +217,7 @@ enum {
 struct {
     opaque dp_config<1..2^16-1>;  /* Encoded differential privacy parameters */
```

Comment:
Reviewer note: it would be slightly more consistent to replace this with `DpConfig` and add the internal length prefix back (it was there before https://github.com/wangshan/draft-wang-ppm-dap-taskprov/commit/f8cedf129c0dfb6a3461b33f90717fcf88cc534b), but I wanted to the minimal change here.